### DISCUSSION TITLE
Specific RelationManager does not show

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#10768] [Help]
I have several relationship managers, but one of them has started not showing up, even though it just started happening.
When I `dd()` the relation it is there

OrderResource
```php
public static function getRelations(): array
{
    return [
        RelationManagers\ServicesRelationManager::class,
        RelationManagers\PassengerRelationManager::class, // Not visible
        RelationManagers\VehicleRelationManager::class,
        RelationManagers\PaymentsRelationManager::class,
    ];
}
```

Order model
```php
/**
 * @var array<int, string>
 */
protected $fillable = [
    'passenger_id',
    // ...
];

public function passenger(): BelongsTo
{
    return $this->belongsTo(
        related: PassengerInfo::class,
        foreignKey: 'passenger_id',
        ownerKey: 'id',
    );
}

// OR

public function passenger(): HasOne
{
    return $this->hasOne(
        related: PassengerInfo::class,
        foreignKey: 'id',
        localKey: 'passenger_id',
    );
}

// EVEN

public function passenger(): HasMany
{
    return $this->hasMany(
        related: PassengerInfo::class,
        foreignKey: 'id',
        localKey: 'passenger_id',
    );
}
```
![7](https://github.com/filamentphp/filament/assets/29732308/f3e431ec-5421-443e-865c-a8119c271575)

**Comment by leandrocfe** [comment]
Did you create it using the command?
https://filamentphp.com/docs/3.x/panels/resources/relation-managers#creating-a-relation-manager

Share the RM code please

**Reply by ArtMin96** [reply]
No. Without command.

```php
class PassengerRelationManager extends RelationManager
{
    protected static string $relationship = 'passenger';

    public static function getTitle(Model $ownerRecord, string $pageClass): string
    {
        return __('passenger.title');
    }

    public function table(Table $table): Table
    {
        return $table
            ->columns([
                // ...
            ]);
    }
}
```

**Reply by ArtMin96** [reply]
It used to work. I just run `php artisan optimize:clear` and it disappeared.

**Reply by leandrocfe** [reply]
You could try to create it using the filament command..

**Reply by ArtMin96** [reply]
Ok. No problem

**Reply by ArtMin96** [reply]
Still not showing

**Reply by ArtMin96** [reply]
```bash
composer du
```
```bash
php artisan optimize:clear
```
Hide any relation manager except this relation manager.

Still not working

**Reply by ArtMin96** [reply]
When I set the name of the relationship incorrectly in the relationship manager, it understands that it is wrong, there is another reason why it is not displaying

**Comment by leandrocfe** [comment]
Can you provide a minimal repo to reproduce it please?

**Reply by ArtMin96** [reply]
Sure

**Reply by ArtMin96** [reply]
I tried it in the Filament demo and there was no such problem.

**Reply by ArtMin96** [reply]
Even when the relation is the same as in the filament demo, the relation manager is the same, again not working. And the most interesting thing is that only that relation manager does not work. Ok. Thanks for your time.
